### DISCUSSION TITLE
Custom formatting for type: "time" axes?

### DISCUSSION THREAD

**Discussion by alexjs-dev** [TanStack/react-charts#92] [General] [answered]
I believe there is no option to format the "time" differently, eg instead of showing the time, showing a data eg. 06/06/2020 or even going more custom with showing weeks number with year, 16/2020. I believe the only option right now type "ordinal"?

**Comment by evenstephenr** [comment]
@fireflypie are you referring to time shown as ticks on an axis, or in a tooltip?

Could you provide a sample codesandbox, such as this one? https://codesandbox.io/s/github/tannerlinsley/react-charts/tree/next/examples/area

**Comment by tannerlinsley** [comment, marked as answer]
Due to a massive overhaul and rewrite of React Charts to TypeScript and a massively upgraded public API, this discussion is being marked as answered for triage. The latest version can be tested via the `react-charts@beta` tag. Documentation is in the process of being upgraded, but thankfully, everything is fully typed now, so you can explore the API that way for now.

If, *after testing or playing with the v3 beta*, you believe this discussion is still relevant or unanswered, feel free to mark it as such and continue the discussion.

**Comment by paulgrieselhuber** [comment]
@tannerlinsley looks like this is indeed still an issue: https://github.com/tannerlinsley/react-charts/issues/261
